src/Controllers/OAuthController.php

```diff
@@ -150,7 +150,7 @@ public function index(Request $request): Response
         if (!$oauth) {
             return $this->redirectRegister(
                 $providerName,
-                $resourceId,
+                (string) $resourceId,
```

Comment:
Can you modify one of the tests to provide a non-string value here? Then it would be covered by a test.